builtin/provisioners/chef-client/ssh_provisioner.go

```diff
@@ -0,0 +1,72 @@
+package chefclient
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/hashicorp/terraform/communicator"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func (p *Provisioner) sshInstallChefClient(
+	o terraform.UIOutput,
+	comm communicator.Communicator) error {
+	var installCmd bytes.Buffer
+
+	// Build up a single command based on the given config options
+	installCmd.WriteString("curl")
```

Comment:
We need to document that curl is a requirement on the base system (it's 2015, it should be on the system, but we should add a note to the docs)